app/views/calendars/index.html.erb

```diff
@@ -1,37 +1,17 @@
 <h1>Calendars</h1>
 <p>Click on show to select a calendar. You can then edit any events on the calendar, or add new ones.</p>
-<p><b>Note:</b> - updating a template calendar does not automatically update any dependent calendars.</p>
+<p><b>Note:</b> - updating a template calendar automatically updates any dependent calendars.</p>
```

Comment:
We don't use the phrase "template calendar" anywhere else in the views. Change to "Area" or "Top level" (or both)?